system_tests/timeboost_test.go

```diff
@@ -45,6 +47,246 @@ import (
 	"github.com/offchainlabs/nitro/util/testhelpers"
 )
 
+func TestExpressLaneTransactionHandlingComplex(t *testing.T) {
```

Comment:
Great tests! Same general comment from me about including a short comment about how the wait group `Add` constants are derived and also looking for a way to get rid of the `time.Sleep`s. (I didn't see a great way to remove them here so it may not be possible.)